src/acceleration/BaseQNAcceleration.cpp

```diff
@@ -666,5 +581,241 @@ void BaseQNAcceleration::writeInfo(
   }
   _infostringstream << std::flush;
 }
+
+void BaseQNAcceleration::concatenateCouplingData(
+    const DataMap &cplData, const std::vector<DataID> &dataIDs, const std::vector<DataID> &primaryDataIDs, Eigen::VectorXd &values,
+    Eigen::VectorXd &oldValues, Eigen::VectorXd &primaryValues, Eigen::VectorXd &oldPrimaryValues)
```

Comment:
```suggestion
void BaseQNAcceleration::concatenateCouplingData(const DataMap &cplData)
```
Could be simplified by using the private members `_dataIDs, _primaryDataIDs, _values...` below.

Reply:
Changes provided via [ee153fa](https://github.com/precice/precice/pull/2005/commits/ee153faa08530c21618bdb63b9569e877cce666f)